main/src/ca/uwaterloo/flix/language/phase/jvm/GenExpression.scala

```diff
@@ -1214,9 +1261,8 @@ object GenExpression {
       mv.visitLabel(ifEnd)
 
     case Expr.Branch(exp, branches, _, _, _) =>
-      // Calculating the updated jumpLabels map
```

Comment:
I think just keep the unrelated code to make the PR prettier (e.g. this comment)